### PR TITLE
Update change log file name

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Change log file name is changed to CHANGELOG.md from history.md but analyze step was looking for history.md and skips if file is missing.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: scripts/devops_tasks/find_change_log.ps1 [hunk 1]
```diff
@@ -15,6 +15,7 @@ function ExtractReleaseNotes($changeLogLocation)
     return $releaseNotes
   }
 
+  Write-Host $changeLogLocation
```
Comment (scbedd) [inline scripts/devops_tasks/find_change_log.ps1]: Debugging? If not and permanent, probably needs a bit of description.

Comment (praveenkuttappan) [reply scripts/devops_tasks/find_change_log.ps1]: Its a debug line and forgot to remove before pushing files.

**Pull request merged**